roles/openshift_repos/tasks/centos_repos.yml

```diff
@@ -19,6 +19,6 @@
     src: "{{ item }}"
     dest: "/etc/yum.repos.d/{{ (item | basename | splitext)[0] }}"
   with_first_found:
-    - "CentOS-OpenShift-Origin{{ ((openshift_version | default('')).split('.') | join(''))[0:2] }}.repo.j2"
+    - "CentOS-OpenShift-Origin{{ ((openshift_release | default('')).split('.') | join(''))[0:3] }}.repo.j2"
```

Comment:
openshift_release is not guaranteed to be defined.  Need to continue using openshift_version here.

Reply:
@michaelgugino can you pls sync up with @vrutkovs and then i'll happily change? And will be good before bot will merge https://github.com/openshift/openshift-ansible/pull/9548

Reply:
Yeah, I probably was wrong - lets revert to `openshift_version`, sorry about that

Reply:
okay @vrutkovs so i guess we should have

`((openshift_version | default('')).split('.') | join(''))[0:3] }}` ? note `.3` vs `.2`